### PR TITLE
Reference inject flags in proxy configuration page

### PR DESCRIPTION
**Pull request by krzysztofdrys**
Overrriding the proxy configuration using the command line flag has been possible for some time.
Update the proxy configuration page to reference that.

Signed-off-by: Krzysztof Drys <krzysztofdrys@gmail.com>

### CODE REVIEW

**Comment by wmorgan** [comment]
Thanks @krzysztofdrys! Looks like lint is complaining about a trailing slash.

File: linkerd.io/content/2.10/reference/proxy-configuration.md [hunk 1]
```diff
@@ -29,6 +29,5 @@ spec:
 See [here](../../tasks/configuring-proxy-concurrency/) for details on tuning the
 proxy's resource usage.
 
-Note that configuration overrides on proxies injected using the `linkerd inject`
-command is planned for release 2.4. Follow this
-[GitHub issue](https://github.com/linkerd/linkerd2/issues/2590/) for progress.
+For proxies injected using the `linkerd inject`, configuration can be overridden
+using the [flags](../cli/inject).
```
Comment (wmorgan) [inline linkerd.io/content/2.10/reference/proxy-configuration.md:33]: How about [command-line flags] ?

**Comment by krzysztofdrys** [comment]
Thanks @wmorgan! I changed the wording and added the trailing slash.

File: linkerd.io/content/2.10/reference/proxy-configuration.md [hunk 2]
```diff
@@ -29,6 +29,5 @@ spec:
 See [here](../../tasks/configuring-proxy-concurrency/) for details on tuning the
 proxy's resource usage.
 
-Note that configuration overrides on proxies injected using the `linkerd inject`
-command is planned for release 2.4. Follow this
-[GitHub issue](https://github.com/linkerd/linkerd2/issues/2590/) for progress.
+For proxies injected using the `linkerd inject`, configuration can be overridden
+using the [command-line flags](../cli/inject/).
```
Comment (alpeb) [inline linkerd.io/content/2.10/reference/proxy-configuration.md:33]:
```suggestion
For proxies injected using the `linkerd inject` command, configuration can be
overridden using the [command-line flags](../cli/inject/).
```

Comment (krzysztofdrys) [reply linkerd.io/content/2.10/reference/proxy-configuration.md:33]: 👍

Comment (krzysztofdrys) [reply linkerd.io/content/2.10/reference/proxy-configuration.md]: I committed and squashed this suggestion.

**Review by alpeb — APPROVED** [review]

**Pull request merged**